Week2/nadin.js

```diff
@@ -0,0 +1,181 @@
+console.log(myString);
+
+// Arrays
+
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+let favoriteMauroAnimals = favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, 'meerkta');
+console.log('i think the new value of the array is:,["blowfish", "meerkta", "capricorn", "giraffe", "turtle"]');
+console.log(favoriteAnimals);
+console.log('The array has a length of:', favoriteAnimals.length);
+const index = favoriteAnimals.indexOf('meerkta');
+favoriteAnimals.splice(index, 1);
+console.log('The item I am looking for is at index:1');
+
+//1
+
+function sumThreeNumber (num1, num2, num3) {
+  return num1+num2+num3;
+}
+const total = sumThreeNumber(1,2,3);
+console.log(total);
+
+//2
+function colorCar (color) {
+return 'a' + ' ' + color + ' ' + 'car';
+}
+const colors = colorCar('red');
+console.log(colors);
+
+//3
+
+const studentObj = {
+    name :"atik",
+    course :"programing",
+    number :'20', 
+}
+
+function printProperties(obj) {
+    for (const key in obj) {
+        console.log(key, ":" , obj[key]);
+      }
+}
+printProperties(studentObj);
+
+//4
+function vehicleType(color, code) {
+   switch (code) {
+       case 1: 
+       console.log('a' + ' ' + color + ' ' +  'car');
+       break;
+       case 2: 
+       console.log('a' + ' ' + color + ' ' +  'motorbike');
+       break;
+       default:
+       console.log('parameter not supported');
+   }
+}
+
+vehicleType("blue", 2);
+
+//5
+3 === 3 ? console.log ('yes') : console.log('no');
+
+//6
+
+function vehicle(color, code, age) {
+    if (age === 5 && code === 1){
+        console.log('a' + ' ' + color + ' ' + 'used car');
+        return;
+    } else if (age === 5 && code === 2){
+        console.log('a' + ' ' + color + ' ' + 'used motorbike');
+        return;
+    }
+    switch (code) {
+        case 1: 
+        console.log('a' + ' ' + color + ' ' + 'car');
+        break;
+        case 2: 
+        console.log('a' + ' ' + color + ' ' + 'motorbike');
+        break;
+        default:
+        console.log('parameter not supported');
+    }
+ }
+
+ vehicle("blue", 1, 5);
+ //a blue used car
+
+ vehicle("blue", 2, 5);
+
+//7 
+const vehicles = ["motorbike", "caravan", "bike"];
+
+//8
+console.log(vehicles[2]);
+
+//9
+function vehicle(color, code, age) {
+    if (age === 1){
+        console.log('a' + ' ' + color + ' ' + 'new' + ' ' + vehicles[code-1]);
+    } else {
+        console.log('a' + ' ' + color + ' ' + 'used' + ' ' + vehicles[code-1]);
+    } 
+ }
+ //10
+const vehicles = ["motorbike", "caravan", "bike"];
+let advertisement = "Amazing Joe's Garage, we service cars, "
+for (let i = 0; i < vehicles.length; i++) {
+     if(i === vehicles.length - 1) {
+         advertisement += "and " + vehicles[i] + "s";
+     } else {
+        advertisement += vehicles[i] + "s, ";
+     }
+ }
+console.log(advertisement);
+
+//11
+const vehicles = ["motorbike", "caravan", "bike"];
```

Comment:
The task is to add a vehicle without changing the code, so while this works there is a better way. Rewrite the task 10 code to take in an array of vehicles and if that works, only `vehicles.push("bus");` should be enough to make a new advertisement.